util-stats/src/main/scala/com/twitter/finagle/stats/StatsReceiver.scala

```diff
@@ -253,4 +256,22 @@ trait StatsReceiver {
   }
 }
 
-abstract class AbstractStatsReceiver extends StatsReceiver
+abstract class AbstractStatsReceiver extends StatsReceiver {
```

Comment:
In order to make overriding varargs from Java source compatible between 2.12.x and 2.13.x, made the varargs methods final, and made a non-varargs non-aliased abstract method with a `scala.collection.Seq` argument as an extension point for overriding.